code/game/machinery/computer/arcade.dm

```diff
@@ -438,12 +461,44 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 	event = null
 	gameStatus = ORION_STATUS_NORMAL
 	lings_aboard = 0
+	killed_crew = 0
 
 	//spaceport junk
 	spaceport_raided = 0
 	spaceport_freebie = 0
 	last_spaceport_action = ""
 
+/obj/machinery/computer/arcade/orion_trail/proc/report_player(mob/gamer)
+	if(gamers[gamer] == -2)
+		return // enough harassing them
+
+	if(gamers[gamer] == -1)
```

Comment:
How is this ever reachable?

Reply:
Line 478

Reply:
You've got a catch 22.  Line 478 is only reachable via the if statement on 475, which... has no way of activating unless 478 has.

Reply:
Here's the full sequence:

1. Trigger the initial psych alert, our tracked score goes from a positive value (3, 4, 5, etc) to -1 on line 494 to show that we're no longer rolling for the alert
2. Kill the crew again. To continue the joke and beg the crew member to stop being unnerving, we issue them a pamphlet. Line 478 then executes and sets our tracked score to -2
3. Continue killing the crew, since our tracked score is -2, we're caught by lines 472/473 and leave them alone. 

I admit it's probably a bit overengineered, but I think the pamphlet is funnier as a "okay well the intervention didn't work, just take this pamphlet for gods sake" thing. I tested it and it functioned correctly in game.